src/__experimental__/components/search/search.style.js

```diff
@@ -194,7 +194,7 @@ export const StyledSearchButton = styled.div`
     margin: 0px 0px;
     padding-bottom: 3px;
     :focus {
-      z-index: ${({ theme }) => theme.zIndex.overlay};
+      z-index: ${({ theme }) => theme.zIndex.smallOverlay};
```

Comment:
I think you should change the other `z-index` in this file to be `smallOverlay` as well